src/QMCApp/ParticleSetPool.cpp

```diff
@@ -271,8 +271,29 @@ ParticleSet* ParticleSetPool::createESParticleSet(xmlNodePtr cur,
     }
     //initialize ions from hdf5
     hid_t h5=-1;
+    //Rather than turn off all H5errors, we're going to
+    //temporarily disable it.
+    //
+    //old_func:  function pointer to current function that
+    //           displays when H5 encounters error.
+
+    herr_t (*old_func)(void*);
+    //old_client_data:  null pointer to associated error stream.
+    void *old_client_data;
+    //Grab the current handler info.
+    H5Eget_auto(&old_func,&old_client_data);
+    //Now kill error notifications.
+    H5Eset_auto(NULL,NULL);
     if(myComm->rank()==0)
       h5 = H5Fopen(h5name.c_str(),H5F_ACC_RDONLY,H5P_DEFAULT);
+    //and restore to defaults.  
+    H5Eset_auto(old_func, old_client_data);
+    if (h5 < 0)
+    {
+      app_error() << "Could not open HDF5 file \"" << h5name
+                  << "\" in ParticleSetPool::createESParticleSet.  Aborting.\n";
+      APP_ABORT("ParticleSetPool::createESParticleSet");
```

Comment:
Since this error may be seen by many users, I suggest adding a comment to try to help them:
"Check file exists and has read permissions"